src/test/java/org/jvnet/winp/NativeAPITest.java

```diff
@@ -85,6 +85,19 @@ public void testGetCommandLine() {
                     continue;
                 }
                 
+                // https://learn.microsoft.com/en-us/windows/win32/debug/system-error-codes--500-999-
+                // This special error code was found for a privileged process on
+                // windows-arm64, coming from Qualcomm driver. Ignore this.
+                if (errorCode == 998) { //ERROR_NOACCESS
+                    continue;
+                }
+
+                // https://learn.microsoft.com/en-us/windows/win32/debug/system-error-codes--0-499-
+                // This happens for some privileged process, skip error.
+                if (errorCode == 299) { //ERROR_PARTIAL_COPY
+                    continue;
+                }
```

Comment:
Is this necessary for this PR, or for adding ARM64 support? If necessary, then I am fine with leaving it here. But if it is only needed for ARM64 support, then maybe we can shrink the diff of this PR.

Reply:
It was unnecessary.